include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -2224,6 +2224,138 @@ namespace internal
     }
   };
 
+
+
+  template <typename Number>
+  struct SimpleVectorDataExchange
```

Comment:
@kronbichler This is a simplified version of `VectorDataExchange` (that does not need a `MatrixFree` object). Since I am using this class in another project, where I run my own matrix-free loops (with own ghosting), I think we could consider to put this into a more central place? Do you have any suggestions?

Reply:
I did not check closely, but could this be shared with the implementation of `internal::VectorDataExchange`? I realize that that class has several additional functions, in particular the capability to identify components within a `MatrixFree` object and to call `update_ghost_values_start` and `_end`, but it seems that combining these might be a good thing to do in the future. I don't think we need to do that now, but looking towards the future it seems this could be in the namespace `Utilities::MPI`, similar to the `Partitioner` class.